recipes-qt/qt5/qtbase_git.bb

```diff
@@ -29,6 +29,7 @@ SRC_URI += "\
     file://0010-linux-clang-Invert-conditional-for-defining-QT_SOCKL.patch \
     file://0011-tst_qlocale-Enable-QT_USE_FENV-only-on-glibc.patch \
     file://0012-mkspecs-common-gcc-base.conf-Use-I-instead-of-isyste.patch \
+    file://0015-Upgrade-double-conversion-to-v3.0.0.patch \
```

Comment:
see the comment above this section.

Reply:
I need to have the patch merged into https://github.com/meta-qt5/qtbase?

Reply:
PR submitted: https://github.com/meta-qt5/qtbase/pull/1

Reply:
I was referring to the comment that these patches are shared between target, native, nativesdk version of qtbase, but you're adding it only to target one.

Reply:
Ah, fixed.